public-ui/src/app/views/survey-view/survey-view.component.html

```diff
@@ -168,6 +171,172 @@ <h2 *ngIf="!loading" class="secondary-display">
                         [backgroundColor]="'#ECF1F4'" [domainType]="'surveys'" [chartTitle]="graphToShow"></app-chart>
                     </div>
                   </div>
+                  <div class="row-expansion" *ngIf="a.expanded && a.subQuestions">
```

Comment:
thanks for doing this. We could probably make this a little cleaner by creating a component.

Reply:
Agree working on pulling this apart into component. Will push the changes soon and let you know when its ready for review again.

Reply:
Pulled out graph-menu and chart into a different component. More to de-duplicated will take care in the next prs.